bridge/cypress/support/pageobjects/EnvironmentPage.ts

```diff
@@ -32,6 +34,16 @@ class EnvironmentPage {
     return this;
   }
 
+  public clickFilterType(stage: string, filterType: string): this {
```

Comment:
Can be a bit improved. We only have the filter types 'problem', 'evaluation' and 'approval' so the parameter can be set to `filterType: 'problem' | 'evaluation' | 'approval'`